### ISSUE TITLE
Rename project

### ISSUE THREAD

**Issue by mrrfv** [mrrfv/linux-android-backup#2]
I think `linux-android-backup` doesn't reflect the project very well, as the script works on all major operating systems (Linux, macOS and Windows via WSL), but I can't think of any other name to use. If you can think of a name, please drop it in the comments.

Requirements:
- can't be too generic or technical - we want to turn this into a grandpa-friendly tool in the future
- needs to be easy to remember

**Comment by Klaasstap** [comment]
Drop (Android) to Desktop

**Comment by a8underscore** [comment]
desktop-android-backup?

**Comment by thefirethirteen** [comment]
Seeing as one of the goals is no "vendor lock-ins or using closed-source software", my suggestions are `open-android-backup`, `libre-android-backup` or `universal-android-backup`.

**Comment by knghtbrd** [comment]
I like libre-android-backup … seems to describe what you're going for here.

**Comment by mrrfv** [comment]
`universal-android-backup` sounds good, it appeals to everyone and makes it clear that the script works on any Android device and operating system. I won't change the name just yet though, as I want to hear everyone's suggestions and opinions.

**Comment by knghtbrd** [comment]
It's your project! You could totally call it android-beep-beep-beep if you wanted to. (Please don't take that as a challenge… I know this is the Internet and all, but still! 🤪)

**Comment by mikedlr** [comment]
Calling it "universal" would suggest also backing up android TVs and similar as well as probably backing up from android to android. I'd only use that name if you think that's part of the plan. Just my little bit of bikeshedding 😀

**Comment by Cam-ak** [comment]
how about "unchained android backup" because your project has no restrictions.

**Comment by mikedlr** [comment]
I like "unchained backup" as a name without the "android" 'cos it will work on other operating systems like LineageOS which are based on ASOP. Call the specific release "unchained backup for android" if that's relevant. It's obvious what it does and it's different enough to remember.

**Comment by upintheairsheep** [comment]
> I like "unchained backup" as a name without the "android" 'cos it will work on other operating systems like LineageOS which are based on ASOP. Call the specific release "unchained backup for android" if that's relevant. It's obvious what it does and it's different enough to remember.

linage os is based on android, and android directives are still android, and will also imply and disappoint users of non-android based platforms looking for backup software.

**Comment by IntoTheVoid-900** [comment]
I think universal-android-backup and unchained-android-backup are a bit too wordy if this is targeting mainstream audiences. Unchained Backup is good, but I do agree with upintheairsheep that it is misleading. Maybe something short and sweet like AndrUp (Android and Backup smashed together)?

**Comment by eshate** [comment]
From my experience, there is always a problem with mapping the real needs. We don't have this problem here, because there are two, simple clues, which should always stay on the top of brainstorm. 
> - can't be too generic or technical - we want to turn this into a grandpa-friendly tool in the future
> - needs to be easy to remember

**My thoughts (and my thoughts only, nothing more):**
- There are _TONS_ of names with _Libre-something, Open-something, Droid-somehow_ - even if it sounds quite straightforward - it will be lost for sure. And really, the next cringe step is "Backupify Droidify", which is horrible.
- The biggest sin (I have that tone because I've worked in this area) would be to give ALL information with the name. It doesn't really work that way - even if it sound logic. Nope, this is rather an oldschool-type thinking, from time when name of your product needed to work as an advertisement also - take "Fresh Burger's made by Tom". Nowadays there's no need for that. Why? Because everything is in **A)** Metadata (which stands for everything around the name, like keywords etc) and **B)** Visuals (there's no need for "Android/Droid" in the name, if there would be an icon/logo with well-known droid). 
- What I'm pointing out is that "BeepBeepBeep"... is actually the best option, from the creative-marketing side of things. Like - seriously. It's original and writes itself into memory. Of course, there could be some enhancements, like "BeepBeepDone" or "Beep&Done" (because it's simple & fast) or "BeepUp" (which sounds like backup, but have other meanings also - and even droid inside!). I am writing this just as an example of thinking. If you really want to make it popular - you should aim more into building meta-dictionary around it with simple, straight visuals. Looking at "unchaineddroidbackup.com" vs "beepup.io" tells us something.
- Which brings me to the next thing - it's not a bad idea to check the domains first. Domains are meaningful for casual end users - "if they managed to get this domain, they need to be serious". But they don't need to know that domain... was first. I just made a quick, 2min run and found that `droup.run` is free. **Dro**(id/back)**up & run** (because it is a script!) - simple & fast. Sincerely - not a mic-drop, but you've got the idea.

**BUT** - if you don't really want to make it much different than it is right now, maybe there's no need for that kind of name. I can see the app in F-Droid right now - "NoRootScriptDroid" - it works. Wait... oh. Last update 4 years ago. 

PS. If you would name the app, for example "Literate Droid" (because the script!) - you could make an icon like this. I'm just putting it for an example. Name is not everything, it should come with neat surroundings, I think. 

<a href="url"><img src="https://www.gameartguppy.com/wp-content/uploads/2018/02/mascot-android_teacher.png" align="left" height="200" width="200" ></a>

**Comment by mrrfv** [comment]
Thanks for all this feedback. I'll rename the project after a big update of some sort - I already have some ideas that'd significantly refine the script, such as:

- separating functionality into their own files to improve maintainability
- creating a proper website and buying a domain for it (this 13kb script will make me bankrupt)
- adding progress bars to the companion app
- making the script easier to use on Windows

Christmas break is about to end for me and I'll be extremely busy again, so it might take some time to make all these changes.

Reddit's opinion on the matter:

![image](https://user-images.githubusercontent.com/92185687/210109371-96627fb3-1854-4cd6-a701-57bb5cd7398f.png)

**Comment by IntoTheVoid-900** [comment]
> Calling it "universal" would suggest also backing up android TVs and similar as well as probably backing up from android to android. I'd only use that name if you think that's part of the plan. Just my little bit of bikeshedding grinning

Well what if you added support for Android to Android backups. In theory most if not all of this script will already work in Termux, so just add a few lines in the README explaining use via Termux.
